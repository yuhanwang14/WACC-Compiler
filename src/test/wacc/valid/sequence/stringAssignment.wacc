# simple string variable declaration and assignment

# Output:

# Program:

begin
  string s = "foo" ;
  s = "bar"
end
