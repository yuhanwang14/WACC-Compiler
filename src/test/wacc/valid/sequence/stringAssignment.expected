 1
 2
 3
 4
 5
 6
 7
 8
 9
10
11
# simple string variable declaration and assignment

# Output:

# Program:

begin
  string s = "foo" ;
  s = "bar"
end
 